templates/_partials/header.tpl

```diff
@@ -0,0 +1,29 @@
+{* StarterTheme: Improve geolocation restriction *}
```

Comment:
As discussed during stand up meeting, to replace with a generic error display method